_docs/_latest/develop/dev-setup.md

```diff
@@ -185,20 +175,30 @@ following (use the values from your terminal, not "`/SOME/PATH/TO`"):
 
 Before you run Forseti, you need to edit the forseti_conf.yaml file, found in
 `forseti-security/configs/forseti_conf.yaml`. Refer to 
-["Configuring Forseti"]({% link _docs/latest/configure/configuring-forseti.md %}) 
+["Configuring Forseti"]({% link _docs/latest/howto/configure/configuring-forseti.md %}) 
 for more information.
 
 ### Executing Forseti commands
 
-After you complete the above steps, you should be able to run the following
-command-line tools:
+After you complete the above steps, you should be able to run the forseti server and the CLI client.
+
+Forseti 2.0 will be on the “dev” branch in github.
+
+Note that these could change, due to 2.0 undergoing rapid change.
```

Comment:
If this is subject to change, please evaluate if you should include. If you can't keep it up to date, let's not include it.

Reply:
I think we have to give people the command to run the server.  So, we should update this as the command changes.